spec/factories/authorization_requests.rb

```diff
@@ -380,9 +380,14 @@
     trait :api_impot_particulier_editeur do
       type { 'AuthorizationRequest::APIImpotParticulier' }
 
+      after(:build) do |authorization_request, _evaluator|
+        next if authorization_request.scopes.any?
+
+        authorization_request.scopes << 'dgfip_annee_n_moins_1' unless AuthorizationRequest::APIImpotParticulier::MANDATORY_REVENUE_YEARS.intersect?(authorization_request.scopes)
```

Comment:
```suggestion
        authorization_request.scopes << 'dgfip_annee_n_moins_1'
```

le `unless` est tout le temps faux car scopes est systématiquement vide